server/src/controllers/internship.ts

```diff
@@ -425,9 +453,10 @@ export async function getAllOperationalAreas(
   } catch (e) {
     return next(e);
   }
-
-  const operationalAreas: string[] = await Internship.distinct("operationalArea");
-
+  const internships = await Internship.find().lean().select("operationalArea");
```

Comment:
Könnte man hier nicht auch wieder das `distinct` mit reinbringen, um direkt in der Datenbank zu filtern und später kein Set draus machen zu müssen?